libraries/src/main/java/com/baeldung/commons/io/FileMonitor.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.commons.io;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.monitor.FileAlterationListener;
+import org.apache.commons.io.monitor.FileAlterationListenerAdaptor;
+import org.apache.commons.io.monitor.FileAlterationMonitor;
+import org.apache.commons.io.monitor.FileAlterationObserver;
+
+import java.io.File;
+
+public class FileMonitor {
+
+	public static void main(String[] args) throws Exception {
```

Comment:
Need to use spaces (not tabs) for indentation. There's a standard formatter for Eclipse that you can import -- see the "eclipse" folder of the tutorials repo.